fs/config.go

```diff
@@ -133,6 +135,8 @@ func NewConfig() *ConfigInfo {
 	c.MultiThreadStreams = 4
 	c.RcJobExpireDuration = 60 * time.Second
 	c.RcJobExpireInterval = 10 * time.Second
+	c.TransferredExpireDuration = 60 * time.Second
```

Comment:
Maybe we could re-use the `RcJobExpireDuration` and `Interval` for this to save on flag explosion?

Reply:
Yeah I had suggestion like that above. It comes down to, if perhaps someone would need to have different schedule for transferred files than with jobs. And since we don't have such requirement, I guess we can reuse them.